docs/Performance.md

```diff
@@ -101,7 +101,7 @@ Similarly, you can implement `shouldComponentUpdate` and indicate the exact cond
 
 "Slow Navigator transitions" is the most common manifestation of this, but there are other times this can happen. Using InteractionManager can be a good approach, but if the user experience cost is too high to delay work during an animation, then you might want to consider LayoutAnimation.
 
-The Animated api currently calculates each keyframe on-demand on the JavaScript thread, while LayoutAnimation leverages Core Animation and is unaffected by JS thread and main thread frame drops.
+The Animated api currently calculates each keyframe on-demand on the JavaScript thread (unless you set `useNativeDriver: true`), while LayoutAnimation leverages Core Animation and is unaffected by JS thread and main thread frame drops.
```

Comment:
I think that braces here are useless and you may remove them. 

Also could you add this as link to this blog post: 
https://facebook.github.io/react-native/blog/2017/02/14/using-native-driver-for-animated.html? 
as it clearly explains why 'unless you set `useNativeDriver: true`' :-)